README.md

```diff
@@ -1,2 +1,103 @@
-# Forms
- building a form in rails
+# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+### Re-former
+
+## In These projects we build some forms, using nearly-pure HTML and using the helper methods that Rails provide.
+
+
+### Prerequisites
+
+- Windows, Mac, or Linux machine 
+
+- Web browser other than Internet Explorer
+
+### Setup
+
+- Install Rails on your computer
+- Clone the repository to a local file
+- Type rails server on your terminal or command line
```

Comment:
Instead of just writing clone the repository you have to show it using command.

```
$ git clone example_repo.git
$ cd example
$ bundle install
$ rails s
```